bitsv/__init__.py

```diff
@@ -3,4 +3,4 @@
 from bitsv.network.services import set_service_timeout
 from bitsv.wallet import Key, PrivateKey, wif_to_key
 
-__version__ = '0.8.0'
+__version__ = '0.9.0'
```

Comment:
Would it not be better to increase the version to 0.9.0 **after** we have fixed everything else? e.g. normalizing the return value of the get_transaction() in APIs etc.
I could be wrong but I thought that you'd want to increase the version number **last thing before cutting a new release** no? (so that when people pin their version, it will include all of the bug fixes?)... or is the tagging and releases not tied to the git commit history like this as I have assumed?

Reply:
@AustEcon GitHub does not automatically create tags from this file, so we can change the version whenever. For clarity for those reading the code, it might be best to increment this right before the release. However, it should not cause any issues for folks downstream if one or two commits are added after the version change and before the actual release.

Reply:
Cool thanks! You learn something everyday! 😆